darwin/exporter/formats/nifti.py

```diff
@@ -135,16 +143,19 @@ def check_for_error_and_return_imageid(video_annotation: dt.AnnotationFile, outp
 
     output_volumes = None
     filename = Path(video_annotation.filename)
-    suffixes = filename.suffixes
-    if len(suffixes) > 2:
-        return create_error_message_json(
-            "Misconfigured filename, contains too many suffixes", output_dir, str(filename)
-        )
-    elif len(suffixes) == 2:
+    try:
+        suffixes = filename.suffixes[-2:]
+    except IndexError:
+        suffixes = filename.suffixes
+    if len(suffixes) == 2:
```

Comment:
Extract the lasst 2 suffixes, which should be `.nii` and `.gz`, but handle the case that there are <2 suffixes

Reply:
Is it still handling case where there are >2 suffixes also?